ProjectName/src/screens/PaymentDetails/PaymentDetailsForm/PaymentDetailsForm.tsx

```diff
@@ -0,0 +1,83 @@
+import {useNavigation} from '@react-navigation/native';
+import React from 'react';
+import {useForm, SubmitHandler} from 'react-hook-form';
+import {Text, View, Pressable} from 'react-native';
+
+import CustomAmountInput from './components/customAmountInput/CustomAmountInput';
+import CustomButton from './components/customButton/CustomButton';
+import CustomInput from './components/customInput/CustomInput';
+import CustomMessageInput from './components/customMessageInput/CustomMessageInput';
+import {styles} from './payment-details-form.styles';
+import {PaymentDetailsFormData} from './payment-details-form.types';
+import {NavigationType} from '../../../navigation/navigation.types';
+
+const PaymentDetailsForm: React.FC = () => {
+  const navigation = useNavigation<NavigationType>();
+
+  const {
+    control,
+    handleSubmit,
+    formState: {errors},
+  } = useForm<PaymentDetailsFormData>({
+    defaultValues: {
+      ToPayee: 'John McCabe',
```

Comment:
Please, be consistent with naming, use `camelCased` variables. So `ToPayee` is better to rename to `toPayee`